docs/references/config/config.toml.md

```diff
@@ -0,0 +1,913 @@
+Example:
+```toml
+seeds = "abcd@1.2.3.4:26656,deadbeef@5.6.7.8:10000"
+```
+
+### p2p.persistent_peers
+Comma-separated list of nodes to keep persistent connections to.
+```toml
+persistent_peers = ""
+```
+
+| Value type                        | string (comma-separated list)           |
+|:----------------------------------|:----------------------------------------|
+| **Possible values within commas** | nodeID@IP:port (`"abcd@1.2.3.4:26656"`) |
+|                                   | `""`                                    |
+
+Persistent peers do not count toward [`p2p.max_num_inbound_peers`](#p2pmax_num_inbound_peers) or
+[`p2p.max_num_outbound_peers`](#p2pmax_num_outbound_peers).
+
+Example:
+```toml
+persistent_peers = "fedcba@11.22.33.44:26656,beefdead@55.66.77.88:20000"
+```
+
+### p2p.addr_book_file
+Path to the address book file.
+```toml
+addr_book_file = "config/addrbook.json"
+```
+
+| Value type          | string                                                      |
+|:--------------------|:------------------------------------------------------------|
+| **Possible values** | relative directory path, appended to `$CMTHOME` |
+|                     | absolute directory path                         |
+
+The default relative path translates to `$CMTHOME/config/addrbook.json`. In case `$CMTHOME` is unset, it defaults to
+`$HOME/.cometbft/config/addrbook.json`.
+
+### p2p.addr_book_strict
+Strict address routability rules disallow non-routable IP addresses in the address book. When `false`, private network
+IP addresses are enabled to be stored in the address book.
+```toml
+addr_book_strict = true
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `true`  |
+|                     | `false` |
+
+Set it to `false` for testing on private network. Most production nodes can keep it at `true`.
+
+### p2p.max_num_inbound_peers
+Maximum number of inbound peers.
+```toml
+max_num_inbound_peers = 40
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt;= 0 |
+
+The [`p2p.max_num_inbound_peers`](#p2pmax_num_inbound_peers) and
+[`p2p.max_num_outbound_peers`](#p2pmax_num_outbound_peers) values together define how many P2P connections the node will
+maintain at maximum capacity, not including list of nodes in the [`p2p.persistent_peers`](#p2ppersistent_peers) setting.
+
+The connections are bidirectional, so any connection can send or receive blocks and other data. The separation into
+inbound and outbound setting only distinguishes the initial setup of the connection: outbound connections are initiated
+by the node while inbound connections are initiated by a remote party.
+
+Nodes on non-routable networks have to set their gateway to port-forward the P2P port for inbound connections to reach
+the node. Outbound connections can be initiated as long as the node has generic Internet access. (Using NAT or methods.)
+
+### p2p.max_num_outbound_peers
```

Comment:
Again on the definition of behavior. The node will try to establish the configured number of outbound peers. While it didn't get this number, it will continuously (although periodically, every 30s) trying to establish connections. So the name maximum maybe is not good. It is more a target.